opentelemetry-user-events-logs/benches/benchmark.rs

```diff
@@ -0,0 +1,34 @@
+// Run as root user as access to `tracefs` is required, or ensure the current user has the necessary permissions.
+// To run benchmarks with root privileges, execute:
+//   sudo -E /home/<username>/.cargo/bin/cargo bench
+// Replace <username> with your actual username.
+//
+// System Information:
+// Processor: AMD EPYC 7763 64-Core Processor
+// CPU Cores: 8
+// Logical Processors: 16
+// Memory: 64 GB
+//  time:   [4.1429 ns 4.1514 ns 4.1621 ns]
```

Comment:
Is 4ns the cost of checking if a particular tracepoint is enabled/disabled?

Reply:
Yes

Reply:
+1 - want to test with concurrent stress from threads.